scripts/views/repoView.js

```diff
@@ -0,0 +1,24 @@
+(function(module) {
+  var repoView = {};
+
+  var ui = function() {
+    var $repos = $('#repos'); 
+    $repos.find('ul').empty();
+    $repos.show().siblings().hide();
+  };
+
+  var render = Handlebars.compile($('#repo-template').html());
+  // var renderFollowers = Handlebars.compile($('#followers-template').html());
```

Comment:
Don't copy-paste code from the pair assignment. I know you don't have to, so don't.